components/central-application-gateway/README.md

```diff
@@ -0,0 +1,109 @@
+# Central Application Gateway
+
+## Overview
+
+This is the repository for the Central Application Gateway.
+
+## Prerequisites
+
+The Central Application Gateway requires Go 1.8 or higher.
+
+## Installation
+
+To install the Central Application Gateway, follow these steps:
+
+1. `git clone git@github.com:kyma-project/kyma.git`
+2. `cd kyma/components/central-application-gateway`
+3. `CGO_ENABLED=0 go build ./cmd/applicationgateway`
+
+## Usage
+
+This section explains how to use the Central Application Gateway.
+
+### Start the Central Application Gateway
+
+To start the Central Application Gateway, run this command:
+
+```
+./applicationgateway
+```
+
+The Central Application Gateway has the following parameters:
+- **disableLegacyConnectivity** is the flag for disabling the default legacy and enabling the Compass mode. The default value is `false`.
+- **proxyPort** is the port that acts as a proxy for the calls from services and Functions to an external solution. The default port is `8080`.
+- **externalAPIPort** is the port that exposes the API allowing to check component status. The default port is `8081`.
+- **application** is the Application name used to write and read information about services. The default Application is `default-ec`.
+- **namespace** is the Namespace in which the Central Application Gateway is deployed. The default Namespace is `kyma-system`.
+- **requestTimeout** is the timeout for requests sent through the Central Application Gateway, expressed in seconds. The default value is `1`.
+- **skipVerify** is the flag for skipping the verification of certificates for the proxy targets. The default value is `false`.
+- **requestLogging** is the flag for logging incoming requests. The default value is `false`.
+- **proxyTimeout** is the timeout for requests sent through the proxy, expressed in seconds. The default value is `10`.
+- **proxyCacheTTL** is the time to live of the remote API information stored in the proxy cache, expressed in seconds. The default value is `120`.
+
+
+## API
+The Central Application Gateway exposes:
+- external API implementing health endpoint for liveness and readiness probes
+- internal API implementing proxy handler accessible via a service of ClusterIP type
+
+### Standalone mode
```

Comment:
Does this mean the legacy mode? We should be consistent with the naming. You use legacy mode in the description of the parameters. 

```suggestion
### Standalone (legacy) mode
```

Reply:
I used the same flag name we already use to distinguish standalone from Compass mode.